Source/Scenes/Overworld.cs

```diff
@@ -483,25 +572,56 @@ public override void Render(Target target)
 						at.X -= width2 + 8 * Game.RelativeScale;
 						UI.Prompt(batch, Controls.Pause, Loc.Str("OptionsTitle"), at, out _, 1.0f);
 					}
-
-					// show version number on Overworld as well
-					UI.Text(batch, Game.VersionString, bounds.BottomLeft + new Vec2(4, -4) * Game.RelativeScale, new Vec2(0, 1), Color.CornflowerBlue * 0.75f);
-					UI.Text(batch, Game.LoaderVersion, bounds.BottomLeft + new Vec2(4, -24) * Game.RelativeScale, new Vec2(0, 1), new Color(12326399) * 0.75f);
-					if (ModLoader.FailedToLoadMods.Any())
-					{
-						UI.Text(batch, string.Format(Loc.Str("FailedToLoadMods"), ModLoader.FailedToLoadMods.Count), bounds.BottomLeft + new Vec2(4, -44) * Game.RelativeScale, new Vec2(0, 1), Color.Red * 0.75f);
-					}
 				}
+			}
 
-				if (cameraCloseUpEase > 0)
-				{
-					batch.PushBlend(BlendMode.Subtract);
-					batch.Rect(bounds, Color.White * Ease.Cube.In(cameraCloseUpEase));
-					batch.PopBlend();
-				}
-				batch.Render(target);
-				batch.Clear();
+			if (cameraCloseUpEase > 0)
+			{
+				batch.PushBlend(BlendMode.Subtract);
+				batch.Rect(bounds, Color.White * Ease.Cube.In(cameraCloseUpEase));
+				batch.PopBlend();
+			}
+
+			var promptPos = bounds.TopCenter + new Vec2(1, 16) * Game.RelativeScale;
+
+			if (selectedModIdx == 0 && state == States.Selecting)
+			{
+				UI.Text(batch, new Loc.Localized("FujiOverworldModSlideNote"), promptPos, new Vec2(0.5f, 0), Color.Gray);
+			}
+			else if (state == States.Selecting)
+			{
+				UI.Text(batch, modsWithLevels[selectedModIdx].ModInfo.Name, promptPos, new Vec2(0.5f, 0), Color.White);
+			}
+		}
+
+		if (Paused && pauseMenu != null)
+		{
+			pauseMenu.Update();
+
+			batch.Rect(bounds, Color.Black * 0.70f);
+
+			pauseMenu.Render(batch, bounds.Center);
+		}
+
+		// show version number on Overworld as well
+		// Logic breakdown:
+		// If paused
+		//  -> Display if the pause menu is in the top-level.
+		// Else
+		//  -> Display if no level is selected.
+		if (Paused ? (pauseMenu is { IsInMainMenu: true }) : (state == States.Selecting))
+		{
```

Comment:
Imo, it makes sense to show the version numbers here, since it makes referencing the version very easy. People tend to skip right past the title screen, and who knows when/if they will pause during gameplay.

Reply:
Oh if it wasn't clear, I meant still showing it on the main overworld display, but just not while it was paused on the overworld.